src/components/AboutUs/StaffCard.js

```diff
@@ -35,57 +33,47 @@ const Description = styled(BodyPrimary)`
   padding-bottom: ${props => remcalc(30)};
 `
 
-const getSocialLink = ({ network, url }) => {
+const SocialLink = ({ name, url, image }) => {
   const size = `${IMAGE_SIZE}px`
 
-  switch (network) {
-    case 'twitter':
-      return (
-        <TappableAnchor href={url}>
-          <Image
-            image={{ file: { url: twitter } }}
-            alt="twitter profile link"
-            width={size}
-            height={size}
-          />
-        </TappableAnchor>
-      )
-    case 'linkedin':
-      return (
-        <TappableAnchor href={url}>
-          <Image
-            image={{ file: { url: linkedin } }}
-            alt="linkedin  profile link"
-            width={size}
-            height={size}
-          />
-        </TappableAnchor>
-      )
-    default:
-      return null
-  }
+  return (
+    <TappableAnchor href={url}>
+      <Image image={image} alt={name} width={size} height={size} />
+    </TappableAnchor>
+  )
 }
 
-const StaffCard = ({ name, image, title, description, socialLinks = [] }) => {
-  const links = (socialLinks || []).map(getSocialLink).filter(e => e)
-  const Links =
-    links.length > 0 ? (
-      <LinksContainer>{links.map((Link, idx) => Link)}</LinksContainer>
-    ) : null
+const SocialLinks = ({ data }) => {
+  return (data || []).length > 0 ? (
+    <LinksContainer>
+      {data.map(({ name, url, image }, idx) => (
+        <SocialLink
+          key={`${name}-${idx}`}
+          name={name}
+          url={url}
+          image={image}
+        />
+      ))}
+    </LinksContainer>
+  ) : null
+}
 
+const StaffCard = ({ name, image, role, description, socialLinks = [] }) => {
   return (
     <Col width={[1, 1, 1, 1, 6 / 12, 4 / 12]}>
-      <Padding bottom={2}>
-        <Image image={image} width="100%" />
-      </Padding>
-      <Subtitle noPadding>{name}</Subtitle>
-      <Padding bottom={3}>
-        <Subtitle noPadding muted>
-          {title}
-        </Subtitle>
+      <Padding bottom={{ smallPhone: 3, smallTablet: 3.5, tablet: 5 }}>
+        <Padding bottom={2}>
+          <Image image={image} width="100%" />
+        </Padding>
+        <Subtitle noPadding>{name}</Subtitle>
+        <Padding bottom={3}>
+          <Subtitle noPadding muted>
+            {role}
+          </Subtitle>
+        </Padding>
+        <Description noPadding>{description}</Description>
+        <SocialLinks data={socialLinks} />
```

Comment:
On mobile, these feel incredibly small, relative to the picture (for example)
![image](https://user-images.githubusercontent.com/15656538/53033583-4b3cd880-3469-11e9-93c8-0f29b5e53f51.png)

Given the discrepancies between the UI Documentation and About Us page version of this component (margin/padding), maybe it's worth double checking with Antonas?

I know you said that you two had a back and forth already. So feel free to ignore this if you feel like you guys have covered it already. Just looks a little off to me, having this quite so small.

Reply:
I'll make sure to mention that do him when he's in the position to do the design review 👍